app.rb

```diff
@@ -87,10 +88,12 @@ def create_book
 
   def create_rental
     puts 'Select a book from the following list by number:'
-    @books.each { |book, i| puts "#{i + 1}) Title: \"#{book.title}\" , Author: #{book.author}"}
+    @books.each { |book, i| puts "#{i + 1}) Title: \"#{book.title}\" , Author: #{book.author}" }
     book_index = gets.chomp.to_i - 1
     puts 'Select a person from the following list by number (not id):'
-    @people.each { |person, i| puts "#{i + 1}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"}
+    @people.each do |person, i| 
+      puts "#{i + 1}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
```

Comment:
- [ ] Kindly verify this function too as after creating a teacher and a book, it does not display when I try to create a rental.
- [ ] Also as I cannot create a rental, I cannot verify the list rental function. Kindly fix your app functionality and make sure everything works as expected 👍